### PR TITLE
Modified cnn

### PR DESCRIPTION
**Pull request by avantikalal**
Modified CNN model to include two conv layers with padding, and variable kernel size. Added more user options including learning rate.

### CODE REVIEW

File: variantworks/networks.py [hunk 1]
```diff
@@ -184,3 +184,73 @@ def forward(self, encoding):
...
+
+        Returns:
+            Instance of class.
+        """
+        super().__init__()
+        self.num_output_logits = num_output_logits
+        self.conv1 = nn.Conv1d(input_feature_size, 128, kernel_size=kernel_size, padding=int((kernel_size-1)/2))
+        self.conv2 = nn.Conv1d(128, 128, kernel_size=kernel_size, padding=int((kernel_size-1)/2))
+        self.gru = nn.GRU(128, gru_size, 1, batch_first=True, bidirectional=True)
+        self.classifier = nn.Linear(2*gru_size, self.num_output_logits)
```
Comment (ohadmo) [inline variantworks/networks.py:234]: Shouldn't the `in_features` for the `nn.Linear` be equal to `gru_size` and not `2*gru_size`?

**Review by ohadmo — CHANGES_REQUESTED** [review]
Looks good to me.
Can you update the `samples/simple_consensus_caller/README.md` since we renamed several files?

File: variantworks/networks.py (hunk 1 above)
Comment (avantikalal) [reply variantworks/networks.py:234]: It's `2*gru_size` since the GRU layer is bidirectional.

**Review by ohadmo — APPROVED** [review]
